swagger.yaml

```diff
@@ -6899,6 +6899,11 @@ definitions:
       file_name:
         type: string
         description: The file name prefaced by the folder that contains it.
+      manifest_source:
+        type: string
+        enum: [user-supplied,system-generated]
+        default: system-generated
+        description: Was this manifest generated by a user or by assisted installer?
```

Comment:
nit: description are usually statement rather than question

Reply:
Fair point, will change this to be a statement